### ISSUE TITLE
saned scan

### ISSUE THREAD

**Issue by takov751** [nextcloud/nextcloud-snap#1048]
Edge case ,but i am really interested. given a working saned service on the server itself,but the nextcloud scanner plugin cant reach it(i presume security feature) ,however is it possible to give access to the needed binary or roll a snap inatance of the saned which could work together ?

**Comment by kyrofa** [comment]
I'm afraid I'm unfamiliar with how that app works, can you break it down for me? It just needs access to a specific binary? Can you tell it where to look?

**Comment by takov751** [comment]
Thank you for answering,sorry i have shut myself out from github recently. Basicly saned  when its installed it can be run as user when you need it , or even you can run it as a systemd service instance(for example when you want to share the scanner on the network)
In this specific case the problem is saned is possible to include in snap, well there is binaries for all architechure ,but its seems unreasonble,because not a real popular addon in nextcloud,however in small company environment comes quiet handy.
 official backend. https://gitlab.com/sane-project/backends/-/releases

the documentation on this page more detailed and straightforward ,than i could write it. down. 
In short saned is in the $PATH ,but it might need configuration or at least config check before running on host ,because the different scanner specification/vendor spec driver.
